templates/Entry/entry.html.twig

```diff
@@ -330,7 +330,7 @@
             {{ entry.content|raw }}
         </article>
 
-        <div class="fixed-action-btn js-fixed-action-btn horizontal click-to-toggle hide-on-large-only">
+        <div class="fixed-action-btn horizontal click-to-toggle hide-on-large-only" data-controller="materialize--fab" data-action="scroll@window->materialize--fab#autoDisplay">
```

Comment:
registering the scroll action this way allows Stimulus handle attaching/detaching the event listener automatically